### PR TITLE
Feature/db setup

### PR DESCRIPTION
**Pull request by joshuaoivie**
Dear Code Reviewer,

This pull request contains the following changes:
1. Setup Database
2. Setup Animals Database
3. Populate the Animals with sample data
4. Perform the prescribed queries
5. Update README.

Here are the query results

Find all animals whose name ends in "mon."

List the name of all animals born between 2016 and 2019.

List the name of all animals that are neutered and have less than three escape attempts.

List the date of birth of all animals named "Agumon" or "Pikachu."

List name and escape attempts of animals that weigh more than 10.5kg.

Find all animals that are neutered.

Find all animals not named Gabumon.

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg).

Warm Regards,
Joshua Ivie.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
+select * from animals where date_of_birth between '2016-01-01' and '2019-12-31';
+select * from animals where neutered = true and escape_attempts < 3;
+select date_of_birth from animals where name = 'Agumon' or name = 'Pikachu';
+select name, escape_attempts from animals where weight_kg > 10.5;
+select * from animals where neutered = true;
+select * from animals where name not like 'Gabumon';
```
Comment (Davidosky007) [inline queries.sql]: * Kindly fix this `select * from animals where name not like 'Gabumon';` query result as when i ran the query this is what i get 
image

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
```
Comment (Davidosky007) [inline queries.sql]: * Kindly fix this query `select * from animals where name like '%mon';` as the result i get is not right 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @joshuaivie 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @joshuaivie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You did well to make the changes requested by the previous reviewer 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**